### ISSUE TITLE
Poor image quality

### ISSUE THREAD

**Issue by Zhentao-Liu** [Janspiry/Image-Super-Resolution-via-Iterative-Refinement#94]
Thanks for you code, it's a great job. I use python sr.py -p val -c config/sr_sr3_64_512.json and your pretrained models to perform super-resolution. But I get poor results which are noisy and blury. step=2000. Could you give some advice?
![0_1_sr](https://user-images.githubusercontent.com/81148025/206690882-8b1a158e-6332-437e-a447-f7ca75b5feaa.png)
![0_2_sr](https://user-images.githubusercontent.com/81148025/206690970-a0f5b538-fa10-4226-b445-8a52f3d5c2ee.png)

**Comment by Lxy98Code** [comment]
> 



> Thanks for you code, it's a great job. I use python sr.py -p val -c config/sr_sr3_64_512.json and your pretrained models to perform super-resolution. But I get poor results which are noisy and blury. step=2000. Could you give some advice? ![0_1_sr](https://user-images.githubusercontent.com/81148025/206690882-8b1a158e-6332-437e-a447-f7ca75b5feaa.png) ![0_2_sr](https://user-images.githubusercontent.com/81148025/206690970-a0f5b538-fa10-4226-b445-8a52f3d5c2ee.png)

Hi, Zhentao. I think your results are reasonable since the pretrained model author shared is limited to `Nvidia 1080Ti`. Enlarge the steps when you train the model, and the results should be better.
I did the same thing as you. However, I found even worse results with same command python sr.py -p val -c config/sr_sr3_64_512.json and pretrained model 'I830000_E32', step=2000. Except for the 'resume state', I didn't change any other config. Do you have any experience about these worse results? Could you share any suggestion? Thanks in advance!

![0_2_sr](https://user-images.githubusercontent.com/61903831/206825218-3ac8260a-0f55-4506-a822-c59807acdd20.png)
![0_2_sr_process](https://user-images.githubusercontent.com/61903831/206825378-f7a14f93-a160-41d4-b956-97cc82589c64.png)

**Comment by Zhentao-Liu** [comment]
> 
Thanks for your reply. You mean I should increase the train step to get better results? Maybe I should try it later. As for your results, it's really wired. I use the same command and the same pretrained model, and I get my results. Isn't there anything wrong? Maybe you should check it more carefully.
![0_2_sr_process](https://user-images.githubusercontent.com/81148025/206826585-f910ca39-62e7-4568-90ae-6d0118850154.png)

**Comment by Lxy98Code** [comment]
Yes, I think so. Take a try to increase the training step for your own model. You can find more details from this issue https://github.com/Janspiry/Image-Super-Resolution-via-Iterative-Refinement/issues/78.

**Comment by Zhentao-Liu** [comment]
> Yes, I think so. Take a try to increase the training step for your own model. You can find more details from this issue #78.

Thanks very much.

**Comment by huchi00057** [comment]
> > 
> 
> > Thanks for you code, it's a great job. I use python sr.py -p val -c config/sr_sr3_64_512.json and your pretrained models to perform super-resolution. But I get poor results which are noisy and blury. step=2000. Could you give some advice? ![0_1_sr](https://user-images.githubusercontent.com/81148025/206690882-8b1a158e-6332-437e-a447-f7ca75b5feaa.png) ![0_2_sr](https://user-images.githubusercontent.com/81148025/206690970-a0f5b538-fa10-4226-b445-8a52f3d5c2ee.png)
> 
> Hi, Zhentao. I think your results are reasonable since the pretrained model author shared is limited to `Nvidia 1080Ti`. Enlarge the iterations when you train the model, and the results should be better. I did the same thing as you. However, I found even worse results with same command python sr.py -p val -c config/sr_sr3_64_512.json and pretrained model 'I830000_E32', step=2000. Except for the 'resume state', I didn't change any other config. Do you have any experience about these worse results? Could you share any suggestion? Thanks in advance!
> 
> ![0_2_sr](https://user-images.githubusercontent.com/61903831/206825218-3ac8260a-0f55-4506-a822-c59807acdd20.png) ![0_2_sr_process](https://user-images.githubusercontent.com/61903831/206825378-f7a14f93-a160-41d4-b956-97cc82589c64.png)

Actually, i had the same trouble like you in my custom weight. But if i executed it on Colab, everything would be perfect.

**Comment by BlouseDong** [comment]
Hello, I have a question for you。 
 “python infer.py -c [config file]”  
What is the config file here? How should I define input low pixel images? 
In other words, I don't know how to conduct testing.

**Comment by 927514606** [comment]
> > > 
> > 
> > 
> > > Thanks for you code, it's a great job. I use python sr.py -p val -c config/sr_sr3_64_512.json and your pretrained models to perform super-resolution. But I get poor results which are noisy and blury. step=2000. Could you give some advice? ![0_1_sr](https://user-images.githubusercontent.com/81148025/206690882-8b1a158e-6332-437e-a447-f7ca75b5feaa.png) ![0_2_sr](https://user-images.githubusercontent.com/81148025/206690970-a0f5b538-fa10-4226-b445-8a52f3d5c2ee.png)
> > 
> > 
> > Hi, Zhentao. I think your results are reasonable since the pretrained model author shared is limited to `Nvidia 1080Ti`. Enlarge the iterations when you train the model, and the results should be better. I did the same thing as you. However, I found even worse results with same command python sr.py -p val -c config/sr_sr3_64_512.json and pretrained model 'I830000_E32', step=2000. Except for the 'resume state', I didn't change any other config. Do you have any experience about these worse results? Could you share any suggestion? Thanks in advance!
> > ![0_2_sr](https://user-images.githubusercontent.com/61903831/206825218-3ac8260a-0f55-4506-a822-c59807acdd20.png) ![0_2_sr_process](https://user-images.githubusercontent.com/61903831/206825378-f7a14f93-a160-41d4-b956-97cc82589c64.png)
> 
> Actually, i had the same trouble like you in my custom weight. But if i executed it on Colab, everything would be perfect.
Hello, I just started to use colab, may I ask how you train in colab, can you share? Thank you very much

**Comment by cementwise** [comment]
> > 
> 
> > Thanks for you code, it's a great job. I use python sr.py -p val -c config/sr_sr3_64_512.json and your pretrained models to perform super-resolution. But I get poor results which are noisy and blury. step=2000. Could you give some advice? ![0_1_sr](https://user-images.githubusercontent.com/81148025/206690882-8b1a158e-6332-437e-a447-f7ca75b5feaa.png) ![0_2_sr](https://user-images.githubusercontent.com/81148025/206690970-a0f5b538-fa10-4226-b445-8a52f3d5c2ee.png)
> 
> Hi, Zhentao. I think your results are reasonable since the pretrained model author shared is limited to `Nvidia 1080Ti`. Enlarge the iterations when you train the model, and the results should be better. I did the same thing as you. However, I found even worse results with same command python sr.py -p val -c config/sr_sr3_64_512.json and pretrained model 'I830000_E32', step=2000. Except for the 'resume state', I didn't change any other config. Do you have any experience about these worse results? Could you share any suggestion? Thanks in advance!
> 
> ![0_2_sr](https://user-images.githubusercontent.com/61903831/206825218-3ac8260a-0f55-4506-a822-c59807acdd20.png) ![0_2_sr_process](https://user-images.githubusercontent.com/61903831/206825378-f7a14f93-a160-41d4-b956-97cc82589c64.png)

您好，我和您遇到了一样的问题，请问您后来解决了吗

**Comment by Alignmentsreies** [comment]
> > > 
> > 
> > 
> > > Thanks for you code, it's a great job. I use python sr.py -p val -c config/sr_sr3_64_512.json and your pretrained models to perform super-resolution. But I get poor results which are noisy and blury. step=2000. Could you give some advice? ![0_1_sr](https://user-images.githubusercontent.com/81148025/206690882-8b1a158e-6332-437e-a447-f7ca75b5feaa.png) ![0_2_sr](https://user-images.githubusercontent.com/81148025/206690970-a0f5b538-fa10-4226-b445-8a52f3d5c2ee.png)
> > 
> > 
> > Hi, Zhentao. I think your results are reasonable since the pretrained model author shared is limited to `Nvidia 1080Ti`. Enlarge the iterations when you train the model, and the results should be better. I did the same thing as you. However, I found even worse results with same command python sr.py -p val -c config/sr_sr3_64_512.json and pretrained model 'I830000_E32', step=2000. Except for the 'resume state', I didn't change any other config. Do you have any experience about these worse results? Could you share any suggestion? Thanks in advance!
> > ![0_2_sr](https://user-images.githubusercontent.com/61903831/206825218-3ac8260a-0f55-4506-a822-c59807acdd20.png) ![0_2_sr_process](https://user-images.githubusercontent.com/61903831/206825378-f7a14f93-a160-41d4-b956-97cc82589c64.png)
> 
> 您好，我和您遇到了一样的问题，请问您后来解决了吗

你好我也遇到类似的问题，请问你是如何解决的？谢谢你的回复
